__tests__/test.spec.js

```diff
@@ -0,0 +1,70 @@
+const percentCalc = require('../database/helper.js');
+let request = require('supertest');
+const app = require('../server/index.js');
+
+let arr = [
```

Comment:
smart to have this mock data. I also did not do this... ![bitmoji](https://sdk.bitmoji.com/render/panel/a3266469-0e26-4d46-953a-2c76c54f5e6d-da971835-7133-40f9-a8c5-6e7e6ca29b2c-v1.png?transparent=1&palette=1&width=246)

I wonder if it would help "clean up" this file if you broke it out into a separate file called "mock data" or whatever that word was that leslie used in class.